src/server/models/flight.js

```diff
@@ -26,4 +31,12 @@ FlightSchema.statics = {
   }
 };
 
-export default mongoose.model('Flight', FlightSchema);
+FlightSchema.statics.getFlightFromFlightName = (flightname) => {
+  const flightNameArray = flightname.split('-');
```

Comment:
The `joi` library for express does this for the URI params as part of the routing validation. You can see this  https://github.com/GoMake/gomake-api/blob/tests-get-telemetry/src/config/param-validation.js#L3

I guess like you I'd expect to see it as part of the controller code or model code more likely; like a static method on the model schema...